include/tsid/contacts/contact-point.hpp

```diff
@@ -101,6 +101,8 @@ namespace tsid
       void setForceReference(ConstRefVector & f_ref);
       void setRegularizationTaskWeightVector(ConstRefVector & w);
 
+      void useLocalFrame(bool local_frame);
```

Comment:
This method should be carefully documented because the implications of switching from local to global frame here are not trivial. For instance, it affects the frame in which the specified contact normal is interpreted.

Reply:
Added comment.